src/lib/containers/works_for_you.tsx

```diff
@@ -128,6 +131,10 @@ export class WorksForYou extends React.Component<Props, State> {
     })
   }
 
+  componentDidUpdate() {
+    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
```

Comment:
`currentScrollOffset?: number = 0` is set to `0` so it's never undefined right ?